### PR TITLE
Add C100 private DNS'

### PR DESCRIPTION
**Pull request by WillTaylor22**

### CODE REVIEW

**Review by timja — CHANGES_REQUESTED** [review]
justice does not need to be in here

File: environments/prod/c100-application-platform-hmcts-net.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+txt:
+
+cname:
+- name: "cdnverify.c100-application"
+  ttl: 300
+  record: "cdnverify.sdshmcts-c100-application-shutter-prod.azureedge.net"
+- name: "c100-application"
+  ttl: 300
+  record: "hmcts-prod.azurefd.net"
```
Comment (timja) [inline environments/prod/c100-application-platform-hmcts-net.yml:15]: this should be in the platform-hmcts-net zone not your own one

File: environments/staging/staging-platform-hmcts-net.yml [hunk 2]
```diff
@@ -34,6 +34,9 @@ A:
     - 10.50.10.117
     ttl: 300
 cname:
+  - name: c100-application
+    ttl: 300
+    record: sdshmcts-stg.azurefd.net
```
Comment (timja) [inline environments/staging/staging-platform-hmcts-net.yml:39]: no need to move this it was already right?

**Review by timja — CHANGES_REQUESTED** [review]

File: environments/staging/staging-platform-hmcts-net.yml (hunk 2 above)
Comment (WillTaylor22) [reply environments/staging/staging-platform-hmcts-net.yml:39]: I had added it, later saw the dupe and deleted the other one. So it looks moved but wasn't the intention.

File: environments/prod/platform-hmcts-net.yml [hunk 3]
```diff
@@ -343,6 +343,12 @@ A:
     ttl: 300
     
 cname:
+  - name: cdnverify.c100-application
+    ttl: 300
+    record: cdnverify.sdshmcts-c100-application-shutter-prod.azureedge.net
```
Comment (timja) [inline environments/prod/platform-hmcts-net.yml:348]:
```suggestion
```

File: environments/prod/platform-hmcts-net.yml [hunk 4]
```diff
@@ -343,6 +343,9 @@ A:
     ttl: 300
     
 cname:
+  - name: c100-application
+    ttl: 300
+    record: hmcts-prod.azurefd.net
```
Comment (timja) [inline environments/prod/platform-hmcts-net.yml:348]:
```suggestion
  - name: c100-application
    ttl: 300
    record: sdshmcts-prod.azurefd.net
```

**Review by timja — APPROVED** [review]

**Pull request merged**